LoopKit/Service/Remote/RemoteActionDelegate.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RemoteActionDelegate.swift
+//  LoopKit
+//
+//  Created by Bill Gestrich on 3/19/23.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
```

Comment:
The `RemoteActionDelegate` is conformed to both by the ServiceManager and DeviceDataManager to propogate actions to where they are delivered.

I'm using "handleRemote" in the method names to help disambiguate from existing APIs in DeviceDataManaager, like "enactBolus"

Reply:
Again, "handle" is very vague. This API is exposing dosing functionality and settings change functionality. So it would be more descriptive to have the verbs describe actually enacting or delivering.  I think it's fine to keep the "remote" in there, as that's the main foreseeable use for this API, but an API like this could theoretically be used by Services for other things, like maybe they have some kind of other trigger to change dosing, like a timer, or other notification.

Reply:
Done but let me know if feedback on the names below.